src/virtual_dom/mod.rs

```diff
@@ -84,13 +84,11 @@ impl Classes {
 
 impl ToString for Classes {
     fn to_string(&self) -> String {
-        let mut buf = String::new();
-        for class in &self.set {
-            buf.push_str(class);
-            buf.push(' ');
-        }
-        buf.pop();
-        buf
+        self.set
+            .iter()
+            .map(String::as_str)
+            .collect::<Vec<&str>>()
```

Comment:
@hgzimmerman could you please add a benchmark to PRs with perf changes like this?

Reply:
How do you suggest that they be integrated into the project?

This one would be easy enough to copy relevant code into a separate project, run the benchmark and throw the project away when done.
Or instead should a benchmarks directory (analogous to tests or examples) be created?
Or finally, just leave a benchmark as part of a tests or benchmarks module within the existing file?

While I await your response, I'll pursue the first option.

Reply:
Yeah I was suggesting running benchmarks locally and pasting the results (ideally in the PR description). I'll add a formal process for this soon